Mathlib/Topology/MetricSpace/Cover.lean

```diff
@@ -66,6 +74,28 @@ nonrec lemma IsCover.of_maximal_isSeparated (hN : Maximal (fun N ↦ N ⊆ s ∧
     IsCover ε s N :=
   .of_maximal_isSeparated <| by simpa [isSeparated_iff_setRelIsSeparated] using hN
 
+/-- A totally bounded set has finite `ε`-covers for all `ε > 0`. -/
+lemma _root_.TotallyBounded.exists_finite_isCover (hs : TotallyBounded s) (hε : ε ≠ 0) :
```

Comment:
```suggestion
lemma exists_finite_isCover_of_totallyBounded (hs : TotallyBounded s) (hε : ε ≠ 0) :
```
for consistency and discoverability, but feel free to make your current name an alias (and similarly for the `IsCompact` version)